APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PurchasesAPI.swift
+//
+//  Created by Madeline Beyl on 8/25/21.
+
+import Foundation
+import RevenueCat
+import StoreKit
+
+func checkPurchasesAPI() {
+    // initializers
+    let purch = Purchases.configure(withAPIKey: "")
```

Comment:
question about our API testers: 
both of them are using abbreviations for variable names. Is there a particular reason for this? We never do abbreviations for any other part of the code (if anything, we have a tendency to go overly verbose at times).

Reply:
I started doing that as shorthand because the api tester variables were never meant to do anything except provide the compiler something. I think short vars when it comes to these tools is ok because we aren't actually using them. If we were to make these projects runnable, or contain any logic, then I would advocate for changing to names to things that make sense. Thoughts?

Reply:
I'm not a fan of abbreviations in any context tbh, they just look odd to me and are harder to read than the actual word. Particularly for lines like [this one](https://github.com/RevenueCat/purchases-ios/pull/839/files#diff-f0a38d89d6f32bb0701941d36f39aeb58b93e3363e6d387e4c805a5f034c4190R24). 
I also feel a bit apprehensive of treating any piece of our code as a second-class citizen, since folks might be tempted to copy / paste or look at our API tester as a reference for API usage
Not a big deal, though.